docs/src/IconButton.doc.js

```diff
@@ -1,419 +1,862 @@
-        icon="angled-pin"
-        {...props}
+        accessibilityLabel="Share pin"
+        icon="share"
+        size="md"
       />
-    )}
-  </Combination>,
-);
-
-card(
-  <Combination id="iconColor" name="Icon Color" iconColor={['darkGray', 'gray', 'red', 'white']}>
-    {(props) => (
       <IconButton
-        icon="angled-pin"
-        accessibilityLabel={`Icon Color ${props.color}`}
-        bgColor={props.iconColor === 'white' ? 'red' : undefined}
-        {...props}
+        accessibilityLabel="Edit pin"
+        icon="edit"
+        size="md"
+      />
+      <IconButton
+        accessibilityControls="selectlist-dropdown-example3"
+        accessibilityExpanded={open}
+        accessibilityHaspopup
+        accessibilityLabel="Menu"
+        icon="ellipsis"
+        onClick={() => setOpen((prevVal) => !prevVal)}
+        ref={anchorRef}
+        selected={open}
+        size="md"
       />
-    )}
-  </Combination>,
+      <Button text="Visit" size="md"/>
+      <Button color="red" text="Save" size="md"/>
+      {open && (
+        <Dropdown anchor={anchorRef.current} id="selectlist-dropdown-example3" onDismiss={() => setOpen(false)}>
+          <Dropdown.Item
+            onSelect={onSelect}
+            option={{ value: "Download image", label: "Download image" }}
+            selected={selected}
+          />
+          <Dropdown.Item
+            badgeText="New"
+            onSelect={onSelect}
+            option={{ value: "Hide Pin", label: "Hide Pin" }}
+            selected={selected}
+          />
+          <Dropdown.Link
+            href="https://pinterest.com"
+            isExternal
+            option={{ value: "Report Pin", label: "Report Pin" }}
+          />
+          <Dropdown.Item
+            onSelect={onSelect}
+            option={{ value: "Delete Pin", label: "Delete Pin" }}
+            selected={selected}
+          />
+        </Dropdown>
+      )}
+    </Flex>
+  )
+}`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Display more than 4 icon buttons in a single row as it can cause cognitive load and usability issues."
+        defaultCode={`
+<Flex gap={2}>
+  <IconButton
+    accessibilityLabel="Click to go back."
+    icon="arrow-back"
+    size="md"
+  />
+  <IconButton
+    accessibilityLabel="Share pin"
+    icon="share"
+    size="md"
+  />
+  <IconButton
+    accessibilityLabel="Edit pin"
+    icon="edit"
+    size="md"
+  />
+  <IconButton
+    accessibilityLabel="Edit pin"
+    icon="add"
+    size="md"
+  />
+  <IconButton
+    accessibilityLabel="Edit pin"
+    icon="search"
+    size="md"
+  />
+  <Button color="red" text="Save" size="md"/>
+</Flex>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Display a [Tooltip](/Tooltip) in conjunction with IconButton to provide context when the IconButton solely would be insufficient."
```

Comment:
"Display a [Tooltip](/Tooltip) in conjunction with IconButton to provide context when the icon alone would be insufficient to convey the purpose of the button."

Reply:
DONE